File: app/src/main/java/za/co/kernelpanic/cloudy/data/ForecastResponse.java
package za.co.kernelpanic.cloudy.data;


import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ForecastResponse {

    @SerializedName("list")
    private List<Forecast> forecastList;
    private City city;


    public City getCity() {
        return city;
    }

    public List<Forecast> getForecastList() {
        return forecastList;
    }

}

File: app/src/main/java/za/co/kernelpanic/cloudy/data/ForecastResponse.kt
package za.co.kernelpanic.cloudy.data

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass (generateAdapter = true)
data class ForecastResponse (val city: City,
                             @Json(name = "list")val forecastList: List<Forecast>)